editor.planx.uk/src/pages/FlowEditor/lib/useScrollControlsAndRememberPosition.ts

```diff
@@ -15,97 +16,96 @@ import { rootFlowPath } from "../../../routes/utils";
 const useScrollControlsAndRememberPosition = (
   scrollContainerRef: React.RefObject<HTMLDivElement>,
 ) => {
+  const currentPath = rootFlowPath(true);
+  const isLoading = useLoadingRoute();
```

Comment:
The fix here is to make these dependencies of the `useEffect()`.

- `currentPath` ensures that each portal and flow retains a scroll position in session storage
- `isLoading` fixes a timing issue to ensure that `scrollTo()` is called once the route has finished loading